Makefile

```diff
@@ -13,7 +13,11 @@ BINDIR ?= $(PREFIX)/bin
 MANDIR ?= $(PREFIX)/share/man
 MAN1DIR ?= $(MANDIR)/man1
 SYSCONFDIR ?= $(PREFIX)/etc
-SYSTEMDUNITDIR ?= $(SYSCONFDIR)/systemd/system
+# Systemd uses a hardcoded unit path list.
+# Sources:
+# - https://github.com/systemd/systemd/issues/19414
+# - https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html
```

Comment:
Remove comment? Of course it does, how could it find them otherwise?

Reply:
Done.